content/vendor/snapshots/snapshots.md

```diff
@@ -0,0 +1,164 @@
+---
+date: 2019-01-23
+linktitle: "Snapshots"
+title: Snapshots
```

Comment:
Would be good to make the title more audience/goal-oriented. "Snapshots" is a name of a feature. This page is about showing vendors how to enable snapshots and recovery options in their KOTS applications.